packages/design-system/.ladle/theme/menu.css.ts

```diff
@@ -35,6 +35,9 @@ export const Menu: ThemeContext['Menu'] = {
                   x: `calc(${sizeVars.v03} + ${sizeVars.v03})`,
                   y: sizeVars.v03,
                 },
+                section: {
+                  y: sizeVars.v03
```

Comment:
Let's remove this (and the other updates to section) and use your fallback approach on the top level Menu instead

Reply:
Since we have the fallback, it feels unnecessary to have the theme value as well

Reply:
good call. changes made.